src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -678,6 +678,37 @@ public function index($columns, $name = null, $algorithm = null)
         return $this->indexCommand('index', $columns, $name, $algorithm);
     }
 
+    /**
+     * Specify a functional index for the table.
+     *
+     * This method allows you to create a functional index using MySQL expressions
+     * like `LOWER`, `UPPER`, or other supported functions directly on columns.
+     *
+     * @param  string|array  $columns  The column(s) to include in the index.
```

Comment:
This could be improved to read:
```suggestion
     * @param  string|string[]  $columns  The column(s) to include in the index.
```